### PR TITLE
Add instructions for Databricks/Snowflake, bump version number

### PR DESCRIPTION
**Pull request by emielver**

### CODE REVIEW

File: content/visualization/visualization_1.md [hunk 1]
```diff
@@ -49,6 +50,20 @@ client_x509_cert_url = "xxx"
...
+access_token = "xxx"
+
+[snowflake]
+user = "xxx"
+password = "xxx"
+account = "xxx"
+database = "xxx"
+schema = "xxx"
+warehouse = "xxx"
+role = "xxx"
```
Comment (rlh1994) [inline content/visualization/visualization_1.md:66]: Should we make clear the snowflake schema should be the derived one?

File: content/visualization/visualization_1.md [hunk 2]
```diff
@@ -9,7 +9,7 @@ Streamlit uses Python to build shareable dashboards without the need for front-e
 Run the command below to download the example dashboard:
 
 ```bash
-git clone --depth 1  --filter=blob:none --sparse https://github.com/snowplow-incubator/snowplow-accelerator-resources.git ; 
+git clone --depth 1  --filter=blob:none --sparse https://github.com/snowplow-incubator/snowplow-accelerator-resources.git ;
 cd snowplow-accelerator-resources
 git sparse-checkout set enhanced-ecommerce-accelerator/snowflake/streamlit
```
Comment (rlh1994) [inline content/visualization/visualization_1.md:14]: How should flag the different options in this command?

File: content/visualization/visualization_1.md (hunk 1 above)
Comment (rlh1994) [inline content/visualization/visualization_1.md:66]: We should add links to some information about where to identify this stuff for each warehouse - asking for a friend.

File: content/visualization/visualization_1.md (hunk 2 above)
Comment (emielver) [reply content/visualization/visualization_1.md:14]: Added tabs like in the docs

**Review by rlh1994 — APPROVED** [review]
LGTM! (once you remove databricks details)

**Pull request merged**